test/mocks/TokenValidFunctions.js

```diff
@@ -0,0 +1,18 @@
+const TokenFunctions = require('../../util/api-utils/token-functions')
+const sinon = require('sinon')
+
+const checkifTokenValidMock = sinon.stub(TokenFunctions, 'checkIfTokenValid')
+
+function restoreMock () {
+  checkifTokenValidMock.restore()
+}
+
+function resetMock () {
+  checkifTokenValidMock.reset()
+}
+
+function setReturnOfTokenValidMock (returnValue) {
```

Comment:
could you name this something that reads more like english? e.g.

```suggestion
function tokenValidWillReturn (returnValue) {
```

Reply:
consider doing this^

Reply:
done